Source/bmalloc/libpas/src/libpas/pas_mte.h

```diff
@@ -65,9 +66,10 @@ PAS_IGNORE_WARNINGS_BEGIN("unsafe-buffer-usage")
  * from OpenSource's pas_internal_config.h -- we cannot use it directly as
  * pas_utils.h is too high up in the include hierarchy.
  */
-#define PAS_MTE_SMALL_PAGE_DEFAULT_SHIFT (14)
```

Comment:
Previously we duplicated this here because we couldn't include pas_internal_config.h, but now we can so we can drop that